example.config.yml

```diff
@@ -73,8 +79,10 @@ installed_extras:
   - xdebug
   - xhprof
 
-# Add any extra apt packages you'd like to install.
+# APT: Add any extra apt packages you'd like to install.
 extra_apt_packages: []
```

Comment:
Let's just change this to `extra_packages`. We can do a minor version bump on the overall project, and people will then be able to choose to update the variable if they're using it already.

Then update the other included files to use `extra_packages`.